src/systemctl/systemctl.c

```diff
@@ -4156,7 +4157,7 @@ static void print_status_info(
                 printf(" %*s %s (%s)\n", 9, t == i->listen ? "Listen:" : "", *t2, *t);
 
         if (i->accept)
-                printf(" Accepted: %u; Connected: %u\n", i->n_accepted, i->n_connections);
+                printf(" Accepted: %u; Connected: %u; Rejected: %u\n", i->n_accepted, i->n_connections, i->n_rejected);
```

Comment:
for compatibility with older releases I'd probably supress the rejected output if it is zero